File: LeetCode/구간 합 1 가장 높은 고도 찾기.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        vector<int> als;
        als.push_back(0);
        int n = 0;
        for (int i : gain) {
            n = n + i;
            als.push_back(n);
        }
        sort(als.begin(), als.end());
        // for (int i : als) {
        //     cout << i << " ";
        // }
        return als[als.size()-1];
    }
};

File: LeetCode/구간 합 1 가장 높은 고도 찾기.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        altitudes = []
        n = 0
        altitudes.append(n)
        for i in gain:
            n += i
            altitudes.append(n)
        altitudes.sort(reverse=True)
        return altitudes[0]